poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
also, yeah I would choose map[string]interface{} over map[string]string here. don't want to limit ourselves to only string values.  future versions of the api should use more appropriate json types.